frontend/src/caseTemplates/empty.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "Empty",
   "description": "N/A",
+  "editable": true,
```

Comment:
Ah, good spot - I had missed that!  (Will have to remember it for the next change!)